cob_lookat_action/CMakeLists.txt

```diff
@@ -15,22 +27,32 @@ generate_messages(
 
 catkin_package(
   CATKIN_DEPENDS actionlib_msgs geometry_msgs message_runtime
+  LIBRARIES cob_lookat_action_server
+  INCLUDE_DIRS include
 )
 
 ### BUILD ###
-include_directories(${catkin_INCLUDE_DIRS})
+include_directories(include ${catkin_INCLUDE_DIRS})
 
-add_executable(cob_lookat_action_server src/cob_lookat_action_server.cpp)
+add_library(cob_lookat_action_server src/cob_lookat_action_server.cpp)
 add_dependencies(cob_lookat_action_server ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 target_link_libraries(cob_lookat_action_server ${catkin_LIBRARIES})
 
+add_executable(cob_lookat_action_server_node src/cob_lookat_action_server_node.cpp)
+add_dependencies(cob_lookat_action_server_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+target_link_libraries(cob_lookat_action_server_node cob_lookat_action_server ${catkin_LIBRARIES})
+
 ### INSTALL ###
-install(TARGETS cob_lookat_action_server
+install(TARGETS cob_lookat_action_server cob_lookat_action_server_node
   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
 )
 
-install(PROGRAMS src/cob_lookat_action_client.py src/cob_monitor_arm_client.py
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}/src
+install(DIRECTORY include/${PROJECT_NAME}/
+  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+)
+
+install(PROGRAMS scripts/cob_lookat_action_client.py scripts/cob_lookat_target_publisher.py
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}/scripts
```

Comment:
I should go into `CATKIN_PACKAGE_BIN_DESTINATION ` directly.
If you code is ptyhon3 compatible, you can try http://docs.ros.org/indigo/api/catkin/html/howto/format2/installing_python.html